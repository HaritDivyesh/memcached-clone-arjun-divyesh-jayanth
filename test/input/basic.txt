# For each command, the first line will be a positive integer W
# where W is the number of lines required by the command.
# This will be followed by W lines of commands
# E.g.
#  _________________
#  | 2             |
#  | set meg 1 0 5 |
#  | Hello         |
#  -----------------
# means the command has 2 writes and expects 1 lines of reply (i.e. STORED)
# Any line beginning with a # is a comment.
# Any empty line is ignored.

1
flush_all
1
get meg
2
set meg 1 0 5
Hello
1
get meg
1
quit

