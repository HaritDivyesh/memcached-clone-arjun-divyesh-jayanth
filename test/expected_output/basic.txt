flush_all
OK
get meg
END
set meg 1 0 5
Hello
STORED
get meg
VALUE meg 1 5
Hello
END
quit
